files/orchestrate/src/sign-payload/app.ts

```diff
@@ -1,13 +1,21 @@
 // tslint:disable: no-console
 import { OrchestrateClient } from "pegasys-orchestrate";
 
+function convertToHex(str: string) {
```

Comment:
Don't we want to use a function from a library like ethers ? No need to rewrite it I think since ethers is already a dependency

Reply:
I tried by ethers appears to only work with numbers